Assignment-4/app/src/Components/SpecificInputValue.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+const SpecificInputValue = () => {
+  const [inputValue, setInputValue] = useState("");
+  const [massage, setMassage] = useState("");
+  const handleClick = () => {
+    if (inputValue === "show") {
+      setMassage("well done friend your input value is correct ");
+    }
+  };
+
+  let result = "";
+  if (massage.length > 0) {
+    result = massage;
```

Comment:
Task 3 not working

Reply:
if we put show input then only massage will display